openg2p_registration/views/openg2p_registration.xml

```diff
@@ -21,20 +21,21 @@
             </field>
         </record>
 
-        <record id="openg2p_registration_view_tree_activity" model="ir.ui.view">
-            <field name="name">openg2p.registration.view.tree.activity</field>
-            <field name="model">openg2p.registration</field>
-            <field name="arch" type="xml">
-                <tree string="Next Activities" decoration-danger="activity_date_deadline &lt; current_date"
-                      default_order="activity_date_deadline">
-                    <field name="name"/>
-                    <field name="activity_date_deadline"/>
-                    <field name="activity_type_id"/>
-                    <field name="activity_summary"/>
-                    <field name="stage_id"/>
-                </tree>
-            </field>
-        </record>
+        <!--below code produced error while upgrading module-->
+<!--        <record id="openg2p_registration_view_tree_activity" model="ir.ui.view">-->
```

Comment:
Can you share the error you were getting from this code?

Reply:
full error message: https://pastebin.com/QiygdQiq